libs/datasets/model_output_schema.py

```diff
@@ -0,0 +1,25 @@
+MODEL_OUTPUT_SCHEMA = [
```

Comment:
It would be great to have this closer to the model that this schema applies to.  or maybe just a comment around which model this actually is.  many different files with the name model in it

Reply:
ranemd to CAN_MODEL_OUTPUT_SCHEMA